tox.ini

```diff
@@ -4,5 +4,10 @@ skipsdist = True
 
 [testenv]
 deps = -rrequirements.test.txt
-commands = pytest --junitxml=test-report.xml --cov=job_engine --cov-report=xml --cov-config=.coveragerc --cov-branch {posargs}
+allowlist_externals =
+    bash
+
+commands =
+    bash -c "ln -sf $(pwd)/nuvla/job_engine {envdir}/lib/python3.11/site-packages/nuvla/"
+    pytest --junitxml=test-report.xml --cov=nuvla --cov-report=xml --cov-config=.coveragerc --cov-branch {posargs}
 
```

Comment:
@ignacio-penas Here is another improvement I'm suggesting. Which allows dynamic determination of the Python version as well.

```
bash -c "ln -sf $(pwd)/nuvla/job_engine {envdir}/lib/python{py_dot_ver}/site-packages/nuvla/"
```

Reply:
@ignacio-penas Thanks for incorporating the suggestion. I'm resolving this conversation.